shortstories/forms.py

```diff
@@ -1,11 +1,10 @@
 from  django import forms
 from .models import ShortStory
-from  django.core import validators
 
 class ShortStoryForm (forms.ModelForm):
 
-    body = forms.CharField(widget =forms.Textarea(attrs={'maxlength':'35000',}))
+    body = forms.CharField(widget=forms.Textarea(attrs={'maxlength':'35000',}))
```

Comment:
we decided to put a min length validation here, remember we postponed this from model task we did earlier.